src/common/ExecutionContext.ts

```diff
@@ -33,7 +33,7 @@ const SOURCE_URL_REGEX = /^[\040\t]*\/\/[@#] sourceURL=\s*(\S*?)\s*$/m;
  *   {@link https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe |
  *   frame } has "default" execution context that is always created after frame is
  *   attached to DOM. This context is returned by the
- *   {@link frame.executionContext()} method.
+ *   {@link frame.executionContext()| method} method.
```

Comment:
this will case the rendered text to read `method method` - the bit after the `|` is the text used. Why did you make this change?

Reply:
@akshat01112001 please could you undo this change?

Reply:
I have undone this change